src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.ts

```diff
@@ -39,18 +48,62 @@ class TeamsReportDeviceUsageDistributionUserCountsCommand extends GraphCommand {
     request
       .get(requestOptions)
       .then((res: any): void => {
-        cmd.log(res);
+        let content: string = '';
+
+        if (args.options.output && args.options.output.toLowerCase() === 'json') {
+          let reportdata = this.getJsonReport(res);
```

Comment:
Please add the type explicitly

Reply:
As we are changing csv rest reponse to json, we made it kind of common for these reports so that we donot need to change even if a new column is introduced. Please let me know if we need to create interface for each report and change this logic. Thanks.

Reply:
Done. This is fixed.